lib/utilities/fastboot-app-module.js

```diff
@@ -0,0 +1,25 @@
+// Expose the an factory for the creating the `Application` object
```

Comment:
I was just refactoring the existing code. :) I plan to do a big cleanup once this is merged, including extracting the testing stuff.